src/api/app/controllers/webui/request_controller.rb

```diff
@@ -436,11 +436,10 @@ def render_request_update
     }
   end
 
+  # TODO: Improve this code. Call require_package instead?
```

Comment:
Does this make this a draft PR? :)

Reply:
No, the comment is there to improve it later. It's no longer a draft PR.

Reply:
Have you created a card in the backlog? :)

Reply:
I'm already working on the improvement. But I prefer to have the quick fix merged and not monkey patched ASAP.